WordPress/Classes/ViewRelated/Post/PostNoticeViewModel.swift

```diff
@@ -125,7 +125,7 @@ struct PostNoticeViewModel {
     }
 
     private var message: String {
-        let title = post.postTitle ?? ""
+        let title = post.titleForDisplay() ?? ""
         if title.count > 0 {
```

Comment:
```suggestion
        if !title.isEmpty {
```

Reply:
Right you are.  Thanks for thinking about it beyond my little change :)